android/src/fdroid/play/listings/ar/release-notes.txt

```diff
@@ -1,13 +1,13 @@
-• New OpenStreetMap data as of May 3
-• Fix search for interpolated odd/even house numbers
-• Fix zoom buttons setting
-• Faster rendering of subway layer
-• Fix wrong postcodes in Paris
-• Fix missing Wikipedia links
-• Fix car routing via roundabouts
-• Enable TTS for Portuguese/Brazil, Español/México
-• Fix location detection over a network on some devices
-• Change gates and Lyon subway icon
-• Fix US road shields
-• Add camera, interior_decoration, antiques, art, cheese shops
+• GPX import support
+• New OpenStreetMap data as of June 2
+• UTM and MRGS coordinates support (tap on coordinates to change)
+• Improved car and bicycle routing
+• Fixed OSM edits upload only on the device's charging
+• Geomagnetic sensor support for devices without gyroscope
```

Comment:
This is confusing.  I think OM already used the magnetic field sensors, but that it required accelerometers to work.   And that now, a device with a gravity sensor and a magnetic sensor, but not a rotation rate sensor, is still able t do some useful fusion.   And that perhaps, the user-facing change is that such devices went from 'no functional compass' to 'compass works'?bn

Reply:
Awesome, thank you for the comments. Fixed.